docs/data/destinations/bigquery.md

```diff
@@ -16,6 +16,17 @@ You can set up recurring syncs of your Amplitude event data to BigQuery through
 
     - [Import data from BigQuery](/data/sources/bigquery)
 
+## Considerations 
+
+- To help save usage costs for your queries, Amplitude partitions the events table by the date of `event_time`. See the [BigQuery docs](https://cloud.google.com/bigquery/docs/partitioned-tables) for more information about table partitioning. 
+- Sometimes duplicate events are exported, especially if you backfill already-exported data. To help with this, Amplitude creates a table function within the provided dataset named `deduplicated_<appid>`. This table function provides a deduplicated view of the data.
```

Comment:
It's not absolutely certain that events are duplicated, do we want to say that its just a possibility?